packages/jellyfish-api-core/__tests__/container_adapter_client.ts

```diff
@@ -17,6 +17,7 @@ export class ContainerAdapterClient extends ApiClient {
   /**
    * Wrap the call from client to testcontainers.
    */
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
   async call<T> (method: string, params: any[], precision: Precision | PrecisionPath): Promise<T> {
```

Comment:
I allow `any` because `call` is a framework code and it is used in a lot of places.
`Unknown` could be better but for now, I keep `any` as I don't want to over refactor the code.